src/http_server.cc

```diff
@@ -2564,6 +2564,12 @@ HTTPAPIServer::EVBufferToRawInput(
         v_idx++;
       }
 
+      if (base_size == 4 && strncmp(base, "Ping", base_size) == 0) {
+        return TRITONSERVER_ErrorNew(
```

Comment:
Are there any resource needed to be released before return? Looks like no need, just want to confirm it with you.